### PR TITLE
2771_Missing_handlers

### PR DESCRIPTION
**Pull request by lilitkarapetyan**
## 🔨Work carried out:

- [x] Tests pass

## Confirmations

- [ ] I have chosen reviewers for my PR.
- [ ] I have assigned myself to this PR.
- [ ] I have chosen an appropriate label for the PR.
- [ ] I have completed the mandatory sections of this document.
- [ ] I have deleted any unused sections.
- [ ] I confirm that I have checked for required README updates and acted accordingly.

### CODE REVIEW

**Comment by IanMayo** [comment]
@lilitkarapetyan - the build is failing with this:
```
-----> Build
       Detected both "build" and "heroku-postbuild" scripts
       Running heroku-postbuild (yarn)
       yarn run v1.22.21
       $ yarn build && yarn build-storybook
       $ cd client && yarn build
       $ rimraf build && vite build && yarn copy-assets && yarn copy-assets-storybook && yarn cleanup
failed to load config from /tmp/build_ac65015b/client/vite.config.ts
error during build:
Error: Cannot find module 'vite-plugin-node-polyfills'
Require stack:
- /tmp/build_ac65015b/client/vite.config.ts
- /tmp/build_ac65015b/client/node_modules/vite/dist/node/chunks/dep-3b8eb186.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1134:15)
    at Module._load (node:internal/modules/cjs/loader:975:27)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/tmp/build_ac65015b/client/vite.config.ts:44:41)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at _require.extensions.<computed> [as .js] (file:///tmp/build_ac65015b/client/node_modules/vite/dist/node/chunks/dep-3b8eb186.js:66199:24)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
error Command failed with exit code 1.
       info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
       info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
       info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
-----> Build failed
       
       We're sorry this build is failing! You can troubleshoot common issues here:
       https://devcenter.heroku.com/articles/troubleshooting-node-deploys
```

File: client/src/ActionsAndReducers/dbWargames/wargames_Reducer.ts [hunk 1]
```diff
@@ -57,9 +57,7 @@ export const wargamesReducer = (state = initialState, action: WargameActionTypes
 
   if (handler) {
     handler(newState, action, tab)
-  } else {
-    console.warn(`Heandler not defined for a action type: ${action.type}`)
-  }
```
Comment (IanMayo) [inline client/src/ActionsAndReducers/dbWargames/wargames_Reducer.ts:62]: Hello @lilitkarapetyan - are you sure that we don't need handlers for these events?

Are they being handled somewhere else?  If so, would it be wise to refer to the handler from this file?

image

**Comment by lilitkarapetyan** [comment]
> @lilitkarapetyan - the build is failing with this:
> 
> ```
> -----> Build
>        Detected both "build" and "heroku-postbuild" scripts
>        Running heroku-postbuild (yarn)
>        yarn run v1.22.21
>        $ yarn build && yarn build-storybook
>        $ cd client && yarn build
>        $ rimraf build && vite build && yarn copy-assets && yarn copy-assets-storybook && yarn cleanup
> failed to load config from /tmp/build_ac65015b/client/vite.config.ts
> error during build:
> Error: Cannot find module 'vite-plugin-node-polyfills'
> Require stack:
> - /tmp/build_ac65015b/client/vite.config.ts
> - /tmp/build_ac65015b/client/node_modules/vite/dist/node/chunks/dep-3b8eb186.js
>     at Module._resolveFilename (node:internal/modules/cjs/loader:1134:15)
>     at Module._load (node:internal/modules/cjs/loader:975:27)
>     at Module.require (node:internal/modules/cjs/loader:1225:19)
>     at require (node:internal/modules/helpers:177:18)
>     at Object.<anonymous> (/tmp/build_ac65015b/client/vite.config.ts:44:41)
>     at Module._compile (node:internal/modules/cjs/loader:1356:14)
>     at _require.extensions.<computed> [as .js] (file:///tmp/build_ac65015b/client/node_modules/vite/dist/node/chunks/dep-3b8eb186.js:66199:24)
>     at Module.load (node:internal/modules/cjs/loader:1197:32)
>     at Module._load (node:internal/modules/cjs/loader:1013:12)
>     at Module.require (node:internal/modules/cjs/loader:1225:19)
> error Command failed with exit code 1.
>        info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
> error Command failed with exit code 1.
>        info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
> error Command failed with exit code 1.
>        info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
> -----> Build failed
>        
>        We're sorry this build is failing! You can troubleshoot common issues here:
>        https://devcenter.heroku.com/articles/troubleshooting-node-deploys
> ```

before `yarn build`, you run the `yarn` commands ?

File: client/src/ActionsAndReducers/dbWargames/wargames_Reducer.ts (hunk 1 above)
Comment (lilitkarapetyan) [reply client/src/ActionsAndReducers/dbWargames/wargames_Reducer.ts:62]: Yes, the handlers for these events are handled in a different part of the codebase.

**Comment by IanMayo** [comment]
> before yarn build, you run the yarn commands ?

That was a Heroku build - so I'm sure it does `yarn install`.

**Comment by lilitkarapetyan** [comment]
> > before yarn build, you run the yarn commands ?
> 
> That was a Heroku build - so I'm sure it does `yarn install`.

I will work on it now.

**Review by IanMayo — APPROVED** [review]
Tested and working

**Pull request merged**